tests/hooks/qamp-participants-conversion-utils.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { beforeEach, describe, expect, test } from "vitest";
+import QampParticipantsAirtableRecords from "../../hooks/qamp-participants-conversion-utils";
```

Comment:
```suggestion
import QampParticipantsAirtableRecords from "~/hooks/qamp-participants-conversion-utils";
```

Reply:
I'm afraid this doesn't work because the when running the tests, we don't have the whole Nuxt context that sets this directory alias, so it fails to find that import.